docs/Common/Enums/index.md

```diff
@@ -1303,3 +1306,16 @@
 |MOBILE|0||
 |CLOUD|1||
 |BOTH|2||
+
+### TransmissionType
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|MANUAL|0|Manual transmission.|
+|AUTOMATIC|1|Automatic transmission.|
+|SEMI_AUTOMATIC|2|Semi automatic transmission.|
+|DUAL_CLUTCH|3|Dual clutch transmission.|
+|CONTINUOUSLY_VARIABLE|4|Continuously variable transmission(CVT).|
+|INFINITELY_VARIABLE|5|Infinitely variable transmission.|
+|ELECTRIC_VARIABLE|6|Electric variable transmission.|
+|DIRECT_DRIVE|7|Direct drive between engine and wheels.|
```

Comment:
```suggestion
|DIRECT_DRIVE|7|Direct drive between engine and wheels.|

```

Reply:
fixed in f87bcddf47dfcb1763618e5d8fdadfd8a17966b5